synapse/lib/storm.py

```diff
@@ -2187,6 +2187,12 @@ def allowed(self, perms, gateiden=None, default=None):
 
         return self.user.allowed(perms, gateiden=gateiden, default=default)
 
+    def reqEditProp(self, prop):
+        if any(self.allowed(perm, gateiden=self.snap.wlyr.iden) for perm in prop.setperms):
+            return
+
+        self.confirm(prop.setperms[-1], gateiden=self.snap.wlyr.iden)
```

Comment:
Worth changing the previous check to `for perm in prop.setperms[:-1]` to avoid a redundant check on the last perm?